README.md

```diff
@@ -1,25 +1,29 @@
 # Microsoft Intune App SDK for Android
 
-Version [11.0.0](https://github.com/msintuneappsdk/ms-intune-app-sdk-android/releases)
+Version [11.0.0](https://github.com/microsoftconnect/ms-intune-app-sdk-android/releases/tag/11.0.0)
```

Comment:
nit: if we update the tag in the URL do we need to update any of our internal docs?